adapters/info.go

```diff
@@ -57,6 +62,11 @@ func (i *InfoAwareBidder) MakeRequests(request *openrtb.BidRequest, reqInfo *Ext
 	// see if any imps need to be removed, and another to do the removing if necessary.
 	numToFilter, errs := i.pruneImps(request.Imp, allowedMediaTypes)
 	if numToFilter != 0 {
+		// If no valid imps left, avoid expensive filtering process
```

Comment:
Can you please reword this comment? It's not avoiding filtering, but rather exiting early, right? We weren't generating an error before this PR.

Reply:
ok